nucypher/characters/control/specifications.py

```diff
@@ -22,11 +22,13 @@ def get_specifications(cls, interface_name: str) -> tuple:
         if cls._specifications is NotImplemented:
             raise NotImplementedError("Missing specifications for character")
         try:
-            input_specification, output_specification = cls.specifications()[interface_name]
+            spec = cls.specifications()[interface_name]
         except KeyError:
             raise cls.SpecificationError(f"{cls.__class__.__name__} has no such control interface: '{interface_name}'")
 
-        return input_specification, output_specification
+        return spec.get('in', tuple()),\
```

Comment:
This bit reads a little awkward on the return line, especially.

Reply:
Yes, we have an "official" standard which is to declare variables before they get returned. right @KPrasch ?

Reply:
Why? Because all `in` should be present in the spec?